package.json

```diff
@@ -114,10 +110,13 @@
     "copyfiles": "^2.0.0",
     "coveralls": "^3.0.2",
     "cross-env": "^5.2.0",
+    "dts-bundle-generator": "^1.6.1",

```

Comment:
I guess after changing the package.json dependencies the jspm section will need to mimic these right?
Do we need jspm as explicit devDependency as well or are you configuring this section by hand?

Reply:
I'm not sure about jspm at all, do we need dev deps for it? Everything doesn't seem to be done with it. Isn't it just for normal dev?

Reply:
well jspm would be a devdep since we don't need it to start/use the plugin